plugins/ibm_qradar/icon_ibm_qradar/util/constants/constant.py

```diff
@@ -0,0 +1,9 @@
+"""Character constants."""
+AQL = "AQL"
+SEARCH_ID = "search_id"
+POLL_INTERVAL = "poll_interval"
```

Comment:
These are not necessary as we use the code-generated Input/Output constants in schema.py files instead.